portfolio/src/main/webapp/src/components/BookService.js

```diff
@@ -1,54 +1,128 @@
 import React, { Component } from "react";
 import Button from "@material-ui/core/Button";
 import { Link } from "react-router-dom";
-import Calendar from 'react-calendar';
+import Calendar from "react-calendar";
 import SearchBar from "./SearchBar";
 
 export default class BookService extends React.Component {
-    constructor() {
-        super();
-        this.state = {
-            date: new Date(),
-            booking_name: "",
-            booking_date: "",
-            booking_duration: "",
-            booking_description: "",
-            booking_price: "",
-        };
-    }
+  constructor() {
+    super();
+    this.state = {
+      booking_name: "Teach Modern Computer Skills to Elderly",
+      //booking_date: new Date(),
```

Comment:
Can you keep both? So we can know what the current date is, and also the booking date.

Reply:
Yes we can! I'm currently working on directly making the calendar input reflect in the datastore!